libbpf-tools/numamove.c

```diff
@@ -3,6 +3,7 @@
 //
 // Based on numamove(8) from BPF-Perf-Tools-Book by Brendan Gregg.
 // 8-Jun-2020   Wenbo Zhang   Created this.
+//30-Jan-2023   Rong Tao      Use fentry_can_attach() to decide use fentry/kprobe.
```

Comment:
Please add a space between '//' and '30-Jan-2023'.

Reply:
Fixed it, thanks!